distributed/cli/dask_scheduler.py

```diff
@@ -21,16 +22,19 @@
 
 
 @click.command()
-@click.option('--port', type=int, default=None, help="Serving port")
+@click.option('--host', type=str, default='',
+              help="URI, IP or hostname of this server")
+@click.option('--port', type=int, default=None, help="Serving port (deprecated, --host is preferred)")
+@click.option('--tls-ca-file', type=str, default=None, help="CA cert(s) file for TLS (in PEM format)")
```

Comment:
This seems good, thanks. Perhaps set the `type` as `type=click.Path` (or `File`) with appropriate args opposed to it being a string?